postgresql.go

```diff
@@ -410,6 +410,11 @@ func (h *postgreSQL) getChecksum(q queryable, tableName string) (string, error)
 }
 
 func (*postgreSQL) quoteKeyword(s string) string {
+	isQuotedColumn := s[0] == '"' && s[len(s)-1] == '"'
```

Comment:
```suggestion
	isQuotedColumn := strings.HasPrefix(s, `"`) && strings.HasSuffix(s, `"`)
```

so there is no empty string problem